kubernetes.yaml

```diff
@@ -51,6 +51,8 @@ spec:
               mountPath: /opt/app/scrapyd_k8s.conf
               readOnly: true
               subPath: scrapyd_k8s.conf
+            #- name: persistent-storage
+            #  mountPath: /data
```

Comment:
Let's name this according to what the persistent storage is used for, so I guess that would be something logging-related, like `logwatch`, `joblogs`, or whatever we decide on.